Add `toYaml`/`fromYaml` policy template functions (#7848)
open-cluster-management/rhacm-docs: release_notes/acm_whats_new.adoc

[#governance-whats-new]
== Governance

> - You can now use the `fromYaml` and `toYaml` Go template functions in `ConfigurationPolicy` to convert YAML strings. You also have more Sprig functions in the list of available functions.

Reviewer: always:

in the `ConfigurationPolicy` resource (or something similar...we don't use the coded `kind` as the noun.

> - You can now use the `fromYaml` and `toYaml` Go template functions in `ConfigurationPolicy` to organize YAML strings. You also have more Sprig functions in the list of available functions.

Author: Rather than "organize", `fromYaml`/`toYaml` perform conversions between strings and objects. Would "convert" be a more appropriate verb?

[#governance-whats-new]
== Governance

> - You can now use the `fromYaml` and `toYaml` Go template functions in `ConfigurationPolicy` to parse YAML strings. More

Reviewer 2: ```suggestion
- You can now use the `fromYaml` and `toYaml` Go template functions in `ConfigurationPolicy` to organize YAML strings. 
```

Author: The `toYaml`/`fromYaml` convert YAML between strings and objects. Would "convert" be a better word?

~And we already have Sprig functions, but we expanded the list of what's available from the Sprig library.~

Author: Oh sorry, that second comment isn't relevant--I hadn't read the second review comment yet! 😅

> Sprig functions are also added to the list of available functions.

Reviewer 2: ```suggestion
You also have more Sprig functions in the list of available functions. 
```
